### PR TITLE
[WIP] add last_update_at to projects

### LINKED ISSUE #1430: Sort project page by commit date
As a user
In order to see which projects have been worked on lately
I would like to order the projects by last worked on date

### PR DESCRIPTION
**Pull request by mattlindsey**
#### Description
Add the last_update_at field to the projects table and populate it.
The next step will be to add a dropdown to allow the user to sort projects by last_update_at.

#### Motivation and Context
It is the first step to allowing user to sort projects.
fixes #1432 

#### How Has This Been Tested?
Tested by running locally and in rspec.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

fixes #1430

### CODE REVIEW

**Comment by mattlindsey** [comment]
Needs an rspec test, at least. @tansaku maybe you can help me tomorrow, since i've never used VCR before.

**Comment by tansaku** [comment]
looks like we can get an updated_at data point from https://developer.github.com/v3/repos/#get

File: db/migrate/20161128165206_add_last_commit_at_to_projects.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddLastCommitAtToProjects < ActiveRecord::Migration
+  def change
+    add_column :projects, :last_update_at, :datetime
```
Comment (tansaku) [inline db/migrate/20161128165206_add_last_commit_at_to_projects.rb]: should we mention that this is from github - not a local thing?

**Comment by tansaku** [comment]
@mattlindsey this is only a piece of #1430 and doesn't actually close that ticket - which requires the front end UX as well - I don't want to merge in stuff in preparation for user stories/acceptance tests.  Let's drive from the acceptance is possible ...

**Pull request closed without merging**